samples/secure_services/src/main.c

```diff
@@ -24,7 +32,7 @@ static void print_hex_number(uint8_t *num, size_t len)
 #endif
 
 
-#if defined(CONFIG_SPM_SERVICE_RNG)
+#if defined(CONFIG_SPM_SERVICE_RNG)  || defined(CONFIG_BUILD_WITH_TFM)
```

Comment:
This service will be enabled unconditionally in TFM? I ask since there is no specific kconfig check for this particular service.